scripts/py_matter_idl/matter/idl/tests/outputs/large_lighting_app/cpp-app/ember-cluster-config/Switch.h

```diff
@@ -0,0 +1,33 @@
+// DO NOT EDIT - Generated file
+//
+// Application configuration for Switch based on EMBER configuration
+// from inputs/large_lighting_app.matter
+#pragma once
+
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/util/cluster-config.h>
+
+#include <array>
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace Switch {
+namespace EmberApplicationConfig {
+
+using FeatureBitmapType = Feature;
+
+inline constexpr std::array<EmberApplicationConfig::ClusterConfiguration<FeatureBitmapType>, 1> kEmberClusterConfig = { {
+    {
+        .endpointNumber = 0,
+        .featureMap = BitFlags<FeatureBitmapType> {
+        },
```

Comment:
This seems... strange.  Switch featuremap always has to have at least one bit set, no?  Is the .zap file here just misconfigured somehow?

Reply:
https://github.com/project-chip/connectedhomeip/blob/master/scripts/py_matter_idl/matter/idl/tests/inputs/large_lighting_app.matter#L1938

In this copy of the test file, the feature map is null

Reply:
How could it possibly be null?  The Switch cluster has _always_ had a feature map, since the first landing.

Reply:
:shrug:  ... this is what we had in the matter file at that time.

Reply:
Lighting app removed this cluster, it has a feature in all clusters now.